### PR TITLE
Update Go modules

### PR DESCRIPTION
**Pull request by loleg**
Removes unnecessary default modules
Removes obselete build steps in CI
Updates Go and JS libraries
Replaces #318

### CODE REVIEW

File: Dockerfile
Comment (loleg) [inline Dockerfile:1]: I just tried to see if the `debian-slim` image starts working again. There are just two warnings:

`debconf: delaying package configuration, since apt-utils is not installed`

and

`rehash: warning: skipping ca-certificates.crt,it does not contain exactly one certificate or CRL`

I think we can undo this change.

**Comment by loleg** [comment]
All tests are passing, I'd appreciate a review and merge  @tafonina

**Review by tafonina — APPROVED** [review]
